src/main/kotlin/lotto/LottoMachine.kt

```diff
@@ -1,20 +1,19 @@
 package lotto
 
 class LottoMachine() {
-    fun createTickets(userAmount: Int): List<Lotto> {
+    fun createGeneratedTickets(
+        totalPurchasedTickets: Int,
+        manualTicketsQuantity: Int,
+    ): List<Lotto> {
         val lottos = mutableListOf<Lotto>()
-        val amountOfTickets = calculateTickets(userAmount)
-        var count = 0
-        while (count < amountOfTickets) {
+        var ticketsToBeGenerated = totalPurchasedTickets - manualTicketsQuantity
+        var countGeneratedTickets = 0
+        while (countGeneratedTickets < ticketsToBeGenerated) {
             lottos.add(Lotto.create())
-            count++
+            countGeneratedTickets++
         }
-        return lottos
-    }
 
-    private fun calculateTickets(userAmount: Int): Int {
-        val numberOfTickets = userAmount / LOTTO_PRICE
-        return numberOfTickets
+        return lottos
```

Comment:
```Kotlin
    val ticketsToBeGenerated = totalPurchasedTickets - manualTicketsQuantity
    return List(ticketsToBeGenerated) { Lotto.create() }
```

If we use Kotlin's power to it's fullest,
this logic can be simplified into above two lines.

Although making logic short isn't always the best option,
it's good to know that Kotlin offers these features for us to use!